libraries/selenium-python37/Dockerfile

```diff
@@ -0,0 +1,19 @@
+
+# PhantomJS for Selenium
+RUN wget https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2 && \
+  tar -xvjf phantomjs-2.1.1-linux-x86_64.tar.bz2 -C /usr/local/share/ && \
+  ln -s /usr/local/share/phantomjs-2.1.1-linux-x86_64/bin/phantomjs /usr/local/bin/
+
+# Chrome for Selenium
+RUN apt-get update && \
+  apt-get install unzip && \
+  wget https://chromedriver.storage.googleapis.com/2.41/chromedriver_linux64.zip && \
+  unzip chromedriver_linux64.zip && \
+  cp chromedriver /usr/local/share/ && \
+  ln -s /usr/local/share/chromedriver /usr/local/bin/ && \
+  DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y  chromium-browser
```

Comment:
at end of this you should add a `&& rm -rf /var/lib/apt/lists/*` (see notes on apt-get installs here: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run)